docs/server.rst

```diff
@@ -32,26 +32,80 @@ the broker.
 
 .. _history: https://github.com/wlanslovenija/tunneldigger/blob/master/HISTORY.rst
 
-Prerequisites
--------------
+Operating System
+----------------
 
 The first thing you need is a recent (>= 2.6.36) Linux kernel that supports L2TPv3
-tunnels. The following modules are required for Tunneldigger operation:
+tunnels.
+
+These distributions are supported:
+
+* Debian
+* Fedora with the package ``kernel-modules-extra``
+* *add your distribution*
+
+Kernel Modules
+--------------
+
+The following modules are required for Tunneldigger operation:
 
 * ``l2tp_core``
 * ``l2tp_eth``
 * ``l2tp_netlink``
 
-In addition the kernel must support network address translation via netfilter,
-otherwise the tunnels will not work as Tunneldigger uses translation to achieve
-that all tunnels operate over the same external port.
+Kernel Module Activation on Boot
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the activation of the kernel modules, we recommend adding a file
+``/etc/modules-load.d/tunneldigger.conf`` with the following content:
+
+.. code:: shell
+
+    l2tp_core
+    l2tp_eth
+    l2tp_netlink
+
+Manual Activation of the Kernel Modules
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can also activate the modules using ``modprobe``.
+A restart will load the kernel without them if you do not create the file.
+
+.. code:: shell
+
+    $ sudo modprobe l2tp_core
+    $ sudo modprobe l2tp_eth
+    $ sudo modprobe l2tp_netlink
+
+Check if Modules are Loaded
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can find out if these modules are loaded by running ``lsmod | grep l2tp``.
+If you get no output, they are not activated.
+If the modules were loaded successfully,
+your listing of the modules might look like this:
+
+.. code:: shell
+
+    $ lsmod | grep l2tp
+    l2tp_eth               16384  0 
+    l2tp_netlink           24576  1 l2tp_eth
+    l2tp_core              32768  2 l2tp_eth,l2tp_netlink
+    ip6_udp_tunnel         16384  1 l2tp_core
+    udp_tunnel             16384  1 l2tp_core
+
+Additional Notes About Kernel Modules
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Also, if you want to have working bandwidth limits, the kernel must support traffic
 shaping and the HTB queuing discipline, together with ``fq_codel``.
 
 The system should be configured to load these modules at boot which is usually done
 by listing the modules in ``/etc/modules``.
```

Comment:
This should be changed to recommend adding them to the file above. Or maybe just add them right there in the docs?

Reply:
(1) Until now, I do not understand this:

> Also, if you want to have working bandwidth limits, the kernel must support traffic
shaping and the HTB queuing discipline, together with ``fq_codel``.

(2) And to me, this

> The system should be configured to load these modules at boot which is usually done
by listing the modules in ``/etc/modules``.

looks like a duplication of the *Kernel Module Activation on Boot* Section and can be removed.

Reply:
fq_codel is the default qdisc for any newer Distribution running systemd. So it's mostly built into the kernel on those distributions. If not it will appear as a kernel module called sch_fq_codel.
Again, I see no need to explicitly state how to create backwards compatibility here since every major distribution has this by default. If you are running a very old and unsupported release of a distribution like Fedora 21, which is now over 4 years old, you probably have a good reason for that and an understanding of how to backport changes to make stuff work. Just mentioning fq_codel in this case will give you enough hints to successfully google a solution to your problem. The tunneldigger documentation shouldn't be the place to teach people how to perform basic Linux administration tasks.
Other than that I do really appreciate the clarity that is being added to the documentation on the other parts. Nice job @niccokunzmann!

Reply:
@kaechele Thanks! I wonder: We could make a section which states what old (4years+) or self-built kernels need in order to run Tunneldigger. If I can read that as a precondition for the section, I can skip this section in normal mode. Would anyone of you appreciate movong the other deleted content to that section?

Reply:
I think removing it is just fine.

Do you want to remove the mention of `/etc/modules` here, or should I merge and then do it?